config/styleguide/assets/animations.css

```diff
@@ -1,4 +1,4 @@
-.animations-disabled * {
+* {
```

Comment:
maybe we should name this file disable-animations.css?

Reply:
Will do.